internal/integration/mysql_test.go

```diff
@@ -502,6 +505,37 @@ schema "test" {
 	})
 }
 
+func TestMySQL_CLI(t *testing.T) {
+	t.Run("Inspect", func(t *testing.T) {
+		myRun(t, func(t *myTest) {
+			defer os.RemoveAll("atlas")
+			c := exec.Command("go", "get", "-d", "ariga.io/atlas/cmd/atlas")
```

Comment:
?

Reply:
I need a way for the integrating test to get this packages on the first time, I moved it to the top of the test.